.github/labeler.yml

```
@@ -0,0 +1,67 @@
+# Common Labels:
+# Add 'oss workflows' label to any change to *.yml files within the workflows directory
+oss workflows:
+  - .github/workflows/*.yml
+
+# Add 'architecture' label to any change to *.yml files within the workflows directory
+architecture:
+  - .github/workflows/*.yml
+
+# Add 'dependencies' label to any change to package.json
+dependencies:
+  - package.json
```

I strongly believe simple lockfixes (changes to `package-lock.json`) should also be included.

```suggestion
dependencies:
  - package*.json
```